fast_api_als/routers/lead_conversion.py

```diff
@@ -26,6 +37,10 @@ def get_quicksight_data(lead_uuid, item):
             Returns:
                 S3 data
     """
+    for id in item:
+        if item[id] == '':
+            logging.error(f'{id} field empty in item')
+            raise Exception(f'{id} field empty in item')
```

Comment:
why?

Reply:
To check that the lead has no properties missing. Should probably change it to warning though.